### DISCUSSION TITLE
Feature request: Add 'quick' search in data explorer

### DISCUSSION THREAD

**Discussion by anbrav0** [posit-dev/positron#4661] [Feedback] [answered]
Hello Posit team!

Thank you all so much for all your wonderful work on the IDE Positron, I have been enjoying it and using it since the public beta launch. This post is for the purpose of a feature request for a 'quick' search method in the data explorer in Positron.

Currently, in RStudio, you are able to quick search through values in the data explorer page like using the magnifying entry (see screenshot):
<img width="1725" alt="image" src="https://github.com/user-attachments/assets/d9fe0dab-4076-4331-a629-5ac6f6517abb">

However currently, in Positron, this feature is not available:
<img width="1437" alt="image" src="https://github.com/user-attachments/assets/4974a195-398f-44bd-a4ee-d0d8dbd03500">

this "quick search" really comes in handy when wrangling data from a biostatistician perspective as it helps to quickly go through/filter values in the data frame, that is small to moderate in size.  I am curious if there is talks in implementing this feature in possible launches!

thank you, and I apologize in advance if there is an issue/discussion already open regarding this! If so, would be happy to be pointed so I may follow and up vote. :-)

**Comment by jmcphers** [comment, marked as answer]
Yes, we've already got an issue open for this! Comment/upvote here: https://github.com/posit-dev/positron/issues/3302

(also some other requests for this from folks working in biostats!)

**Reply by anbrav0** [reply]
Thank you so much for this! I will go ahead and close this discussion in favor of following #3302 for updates related to this!
